apollo-client/src/main/java/com/ctrip/framework/apollo/model/ConfigChangeEvent.java

```diff
@@ -55,4 +97,11 @@ public boolean isChanged(String key) {
   public String getNamespace() {
     return m_namespace;
   }
+
+  /**
+   * @return all changed keys and instances
+   */
+  public Map<String, ConfigChange> getChanges() {
```

Comment:
there were users asking for this api but I thought it's not necessary since we could combine `changedKeys` and `getChange` to achieve this effect.

Reply:
get it